src/sentry/event_manager.py

```diff
@@ -1624,6 +1624,17 @@ def _save_aggregate(
 
                 group = _create_group(project, event, **kwargs)
 
+                if features.has(
+                    "projects:first-event-severity-calculation", event.project
+                ) and not group.data.get("metadata", {}).get("severity"):
```

Comment:
It can, but if I'm correct about the source of the bug, zero-scores won't make it this far in any case. If I'm wrong, then the false positives here will actually also be a useful signal. (And then once the bug is fixed, this bit will come out.)